.github/workflows/check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+"on":
```

Comment:
> `"on"`

I'm not a fan of this particular change but I get it and it's obviously not a deal breaker. Thank you for the PR!

Reply:
We can revert it. It's an option for the linter. I'll do it.